modules/core/src/utils/index.ts

```diff
@@ -21,6 +21,7 @@ export * from "./certificate-management-utils";
 export * from "./common-utils";
 export * from "./context-utils";
 export * from "./crypto-utils";
+export * from "./encode";
 export * from "./encode-decode-utils";
```

Comment:
Wouldn't having two files with `encode` term cause confusion? Also, the term `encoding` doesn't correctly convince what we are doing here.

IMO, it'd better to use a term such as `sanitize`, `escape` etc.

Reply:
cc @brionmario WDYT ?